jekyll/_cci2/contexts.md

```diff
@@ -32,21 +33,9 @@ workflows:
             context: org-global
 ```
 
-## Environment Variables in Contexts
-Contexts have a set of environment variables defined as name/value pairs. When a job is run with a Context the environment variables of the Context are injected at runtime. The order of precedence is important to remember when working with environment variables that might overlap in name. Following is the order, with later injection always taking precedence when the steps of your job run:
+**Note:** Environment variables are considered in a specific order, as follows:
 1. Project-level environment variables set on the Project Settings page.
 2. Context environment variables (assuming the user has access to the Context).
-3. Environment variables set in your config YAML with the `environment` key of a job definition.
+3. Environment variables set in the `config.yml` file with the `environment` key of a job definition.
 4. Environment variables declared inside a shell command in a `run` step of a job.
 
-## Creating and Editing Contexts
-You can manage your organization's Contexts from the Organization Settings page. 
-
-To create a new context hit the "Add Context" button from the Contexts page.
-
-You can edit existing Contexts by clicking them in the list on the Context page.
-
-## Context Security
```

Comment:
I'd rather this go into a blog post or on discuss itself, when we have a new feature or piece of functionality, we add it to the docs.